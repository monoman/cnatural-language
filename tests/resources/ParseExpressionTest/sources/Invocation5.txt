F(G < A, B >> 7)