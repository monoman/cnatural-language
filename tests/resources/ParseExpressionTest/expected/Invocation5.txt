Name(F)((Name(G) LessThan Name(A)), (Name(B) RightShift Literal(DecimalInteger, [7])))